pkgs/by-name/ij/ijq/package.nix

```
@@ -1,21 +1,37 @@
-{ buildGoModule, fetchFromSourcehut, lib, jq, installShellFiles, makeWrapper, scdoc }:
+{
+  buildGoModule,
+  fetchFromSourcehut,
+  lib,
+  jq,
+  installShellFiles,
+  makeWrapper,
```

```suggestion
  makeBinaryWrapper,
```

For 2 reasons: a small performance gain and a better compatibility on macOS.